.github/workflows/ci_tests.yaml

```
@@ -79,7 +81,10 @@ jobs:
 
       # Install GMT and other required dependencies from conda-forge
       - name: Install dependencies
-        run: conda env update --file environment.yml
+        run: |
+          conda install gmt=6.1.1 numpy pandas xarray netCDF4 packaging \
+                        codecov coverage[toml] ipython make \
+                        pytest-cov pytest-mpl pytest>=6.0
```

Do we want to remove the pytest>=6.0 restriction? We pinned to >=6.0 at #725, apparently because only pytest=6.0 has pyproject.toml support. Pytest is at 6.2.2 now (two minor versions ahead of 6.0) so probably less likely for someone to install pytest < 6?

```suggestion
                        pytest pytest-cov pytest-mpl
```
If we want to remove the pin, then please change environment.yml to remove the pytest pinning too.

---

pytest 6.0.0 was released less than 1 year ago. I feel it would be safer to keep the `>=6.0` version restriction for a longer time.

---

Ok, fair enough!